locales/fr/learn.ftl

```diff
@@ -3,7 +3,9 @@
 learn-page-title = Apprendre Rust
 learn-begin = Commencer avec Rust
 learn-book = Affectueusement surnommé le "Book", <cite>The Rust Programming Language</cite> vous donnera une vue d'ensemble du langage depuis ses premiers principes. Vous construirez quelques projets au long du chemin, et à la fin, vous aurez une solide compréhension du langage.
-learn-book-button = Lire le Book !
+learn-book-button = Lire le Book officiel !
```

Comment:
I would still just call it `le Book` here

Reply:
My feeling was that a nice warning was needed before a change of language. But I can remove "officiel" if you want.